packages/base/src/asset-registries/Illustrations.js

```diff
@@ -13,18 +14,58 @@ const registerIllustration = (name, { dialogSvg, sceneSvg, spotSvg, set, title,
 	});
 };
 
+const registerIllustrationLoader = async (illustrationName, loader) => {
+	loaders.set(illustrationName, loader);
+};
+
+const _loadIllustrationCollectionOnce = async illustrationName => {
+	let temp;
```

Comment:
`temp` is not a good variable name as it is too generic, please choose a more descriptive name

Reply:
Thanks for pointing out, is `temporaryVariable` better ?